### PR TITLE
Short circut for ipfs-proxy

### PR DESCRIPTION
**Pull request by crazybuster**
First pull request? Read our [guide to contributing](https://docs.originprotocol.com/guides/getting_started/contributing.html)

### Description:

Please explain the changes you made here:

- A description of the problem you're trying to solve
- An overview of the suggested solution
- If the feature changes current behavior, reasons why your solution is better

### Checklist:

- [X] Test your work and double-check to confirm that you didn't break anything
- [ ] [Wrap any new text/strings with `fbt`](https://github.com/OriginProtocol/origin/tree/master/dapps/marketplace/translation#wrapping-text) for translation
- [ ] If there are any new/changed strings to translate, [`npm run translate`](https://github.com/OriginProtocol/origin/tree/master/dapps/marketplace/translation#integrating-translations)
- [ ] Update any relevant READMEs and [docs](https://github.com/OriginProtocol/origin/tree/master/docs)

### CODE REVIEW

File: infra/ipfs-proxy/src/app.js [hunk 1]
```diff
@@ -39,6 +42,94 @@ function isValidVideo(buffer) {
   return file && validVideoTypes.includes(file.mime)
 }
 
+function getMultiHash(buffer) {
+  const unixFs = new Unixfs("file", buffer)
+  return new Promise((resolve, reject) => {
+    dagPB.DAGNode.create(unixFs.marshal(),
```
Comment (mikeshultz) [inline infra/ipfs-proxy/src/app.js]: Can't find this in the docs or source for `js-ipld-dag-pb`. Older version?  This PR tested?

**Comment by franckc** [comment]
Sorry I haven't reviewed yet but I did not forget... it is on my TODO list. Thanks for taking a stab at it... I hope we won't need it but it makes me feel better that we have a fallback in case our IPFS server completely melts down...

**Comment by franckc** [comment]
We chatted about this PR during the IPFS outage post-mortem.
The recommendation was to keep this PR open and to not merge it until we really need it.
CC @tomlinton @mikeshultz

**Comment by tomlinton** [comment]
Lets close it and reopen if required to clean up the PR queue.

**Pull request closed without merging**